feedaty.php

```diff
@@ -594,7 +594,7 @@ public function fdGetProductData($id)
 	private function fdGenerateStars($data)
 	{
 		if (!isset($data)) $data = 0;
-		return '<img src="http://'.'www.feedaty.com/rating/rate-small-'.(int)$data.'.png" height="15">';
+		return '<img src="//'.'widget.zoorate.com/rating/rate-small-'.(int)$data.'.png" height="15">';
```

Comment:
The concat between `//` and `widget.zoorate` is not useful and should be removed.